k8s/configuration/secret.yaml

```diff
@@ -14,9 +14,9 @@ metadata:
   namespace: default
 type: Opaque
 data:
-  # Base64-encoded Group ID
-  # Group ID used by MongoDB deployment
-  group-id: "<b64 encoded Group ID>"
+  # Base64-encoded Project ID
+  # Project ID used by MongoDB deployment
+  group-id: "<b64 encoded Project ID>"
```

Comment:
why not change this key to `project-id` as well?

Reply:
Because internally it is still `group-id` in the configuration file.
So, I would maybe change it later when MongoDB starts using `project id` as a parameter in the configuration file.